client/app/bundles/KudosApp/components/KudosList.jsx

```diff
@@ -82,7 +72,10 @@ export default class KudosList extends React.Component {
           updateKudo={updateKudo}
           isFetchingKudos={isFetchingKudos}
         />
-        {isFetchingKudos ? <Spinner /> : null}
+        <div className="kudos-list__fetching-container">
+          {isFetchingKudos ? <Spinner /> : <LoadMore onClick={this.loadMoreKudos} />}
+        </div>
+        <BottomScrollListener debounce={0} offset={300} onBottom={this.loadMoreKudos} />
```

Comment:
@joecox in response to your other comment, I tried setting `debounce` to zero and performance seems much better (since it will call `loadMoreKudos` immediately after recognizing bottom of window